massa-network/src/peer_info_database.rs

```diff
@@ -17,22 +15,41 @@ use tokio::task::JoinHandle;
 use tokio::time::{sleep, Duration};
 use tracing::{trace, warn};
 
+/// Peer categories.
+/// There is a defined number af slots for each category.
+#[derive(Clone, Copy, Serialize, Deserialize, Debug, PartialEq, Eq)]
+pub enum PeerType {
+    /// if the peer is in bootstrap servers list
+    /// for now it is decoupled from the real bootstrap sever list, it's just parsed
+    /// TODO: https://github.com/massalabs/massa/issues/2320
+    Bootstrap,
+    /// Connection from these nodes are always accepted
+    WhiteListed,
+    /// Just a peer :pear:
```

Comment:
i's pronounced quite differently :p

Reply:
It sounds a little bit more similar in british english :p 

Anyway do we keep the joke ? :cactus:

Reply:
I would remove this one because it's not flattering to our English skills :p